site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -22,16 +22,44 @@ const createThread = (title, content, category) => {
 };
 
 const replyToThread = (title, reply) => {
-    cy.get('.thread-left-cont > .thread-list-item').contains(title).click();
-    cy.get('.create-post-head').should('contain', title);
+    cy.get('[data-testid="thread-item"]').contains(title).click();
+    cy.get('[data-testid="post-head"]').should('contain', title);
     cy.get('#reply_box_3').type(reply);
     cy.get('[value="Submit Reply to All"]').should('not.be.disabled').click();
     cy.get('#posts_list').should('contain', reply);
 };
 
+const upduckPost = (thread_title) => {
+    cy.get('[data-testid="thread-item"]').contains(thread_title).click();
+    cy.get('[data-testid="post-head"]').should('contain', thread_title);
+    cy.get('.first_post > [data-testid="like-count"]').should('have.text', 0);
```

Comment:
Do you need .first_post for this ?

Reply:
yes, otherwise it will also read the reply's like-count and combine them making the text "00"

Reply:
Changed to using first like count the post sees.